src/control/projectControl.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-undef */
+/* eslint-disable no-shadow */
+/* eslint-disable no-empty */
```

Comment:
You should really try to do your best to adopt the style provided, disabling so many rules on each file is kind of cheating 😅.

Reply:
Noted. I've edited the code to satisfy eslint.